core/src/main/java/com/github/phantazmnetwork/core/config/processor/MinestomConfigProcessors.java

```diff
@@ -50,6 +60,70 @@ public Potion dataFromElement(@NotNull ConfigElement element) throws ConfigProce
             return node;
         }
     };
+    private static final ConfigProcessor<Point> POINT = new ConfigProcessor<>() {
+
+        private final ConfigProcessor<Vec3D> vectorProcessor = VectorConfigProcessors.vec3D();
+
+        @Override
+        public @NotNull Point dataFromElement(@NotNull ConfigElement element) throws ConfigProcessException {
+            return VecUtils.toPoint(vectorProcessor.dataFromElement(element));
+        }
+
+        @Override
+        public @NotNull ConfigElement elementFromData(@NotNull Point point) throws ConfigProcessException {
+            return vectorProcessor.elementFromData(VecUtils.toDouble(point));
+        }
+    };
+    private static final ConfigProcessor<NBT> NBT = new ConfigProcessor<>() {
+        @Override
+        public @NotNull NBT dataFromElement(@NotNull ConfigElement element) throws ConfigProcessException {
+            String nbtString = ConfigProcessor.STRING.dataFromElement(element);
+            NBT nbt;
+            try {
+                nbt = new SNBTParser(new StringReader(nbtString)).parse();
+            }
+            catch (NBTException e) {
+                throw new ConfigProcessException(e);
+            }
+
+            return nbt;
+        }
+
+        @Override
+        public @NotNull ConfigElement elementFromData(@NotNull NBT compound) {
+            return new ConfigPrimitive(compound.toSNBT());
+        }
+    };
+    private static final ConfigProcessor<VillagerMeta.VillagerData> VILLAGER_DATA = new ConfigProcessor<>() {
```

Comment:
any reason there is a processor specifically for villagermeta and no others

Reply:
to cover each meta type

Reply:
but this is only for villagermeta xd

Reply:
there exists a setX(VillagerMeta.VillagerData data) method somewhere so this covers that for reflection purposes